plugin/core/sessions.py

```diff
@@ -371,21 +447,31 @@ def _handle_initialize_result(self, result: Any) -> None:
             debug("session with no workspace folders")
 
         self.state = ClientStates.READY
-
-        self.on_request("workspace/workspaceFolders", self._handle_request_workspace_folders)
-        self.on_request("workspace/configuration", self._handle_request_workspace_configuration)
-        self.on_request("client/registerCapability", self._handle_register_capability)
-        self.on_request("client/unregisterCapability", self._handle_unregister_capability)
+        self.on_initialized()
         if self.config.settings:
-            self.client.send_notification(Notification.didChangeConfiguration({'settings': self.config.settings}))
+            self.send_notification(Notification.didChangeConfiguration({'settings': self.config.settings}))
+        mgr = self.manager()
+        if mgr:
+            mgr.on_post_initialize(self)
```

Comment:
Can use `call_manager`?

Reply:
I actually want to get rid of `call_manager`. It's a temporary solution. Any calls to the `Manager` should use the abstract base class.